core/server/web/shared/middlewares/cache-clear.js

```diff
@@ -0,0 +1,32 @@
+// # CacheClear Middleware
+// Usage: Automatic
+// After: cacheControl
+// Before: routes
+// App: Admin|Site|API
+//
+// Allows deletion of files stuck in cache
+const fs = require('fs-extra');
+const common = require('../../../lib/common');
+
+const deleteSingleFile = file => fs.unlink(file.path).catch(err => common.logging.error(err));
+
+const cacheClear = (req, res, next) => {
+    const delFiles = () => {
```

Comment:
The rule we are trying to keep is preferring long form variable naming to short ones. So, to be consistent with a method name above `deleteSingleFile` would go with  `deleteFiles` here :+1: